src/popup/utils/constants.ts

```diff
@@ -23,14 +23,17 @@ export const AETERNITY_SYMBOL = 'AE';
 export const AETERNITY_COIN_SYMBOL = 'AE Coin';
 export const AETERNITY_COIN_NAME = 'Aeternity';
 
+export const TX_DIRECTION = {
+  credit: 'sent',
+  debit: 'received',
```

Comment:
I spoke with Subhod and Badi and I feel that using "credit" / "debit" makes sense in general, but not in our case. The SDK is using "sent" and "receive" so we (Badi and me) think that it should be "sent" / "receive".